src/Service/FirstRegistrationService.php

```diff
@@ -312,8 +312,12 @@ public function zgwToFirstRegistrationHandler(array $data, array $configuration)
         }
 
         // Loop through the
-        $documents = [];
-        foreach ($zaakObject->getValue('zaakinformatieobjecten') as $zaakInfoObject) {
+        $documents              = [];
+        $zaakInfoObjectSchema   = $this->resourceService->getSchema('https://vng.opencatalogi.nl/schemas/zrc.zaakInformatieObject.schema.json', 'common-gateway/zds-to-zgw-bundle');
+        $zaakInformatieObjecten = $this->cacheService->searchObjects(null, ['embedded.zaak.identificatie' => $identification], [$zaakInfoObjectSchema->getId()->toString()])['results'];
+
+        foreach ($zaakInformatieObjecten as $zaakInfoArray) {
+            $zaakInfoObject   = $this->entityManager->getRepository(ObjectEntity::class)->find($zaakInfoArray['_id']);
```

Comment:
Note: the ObjectEntity::class way of getting repositories is the preferred way in Symfony 6/Doctrine 3, 'App:ObjectEntity' is not supported in those versions.

Reply:
Did you find a way to fix this in CoreBundle InstallationService btw?

Reply:
Working with 'App\Entity\ObjectEntity' would also work, but is less reliable when something changes namespace.